customersupport/models.py

```diff
@@ -252,3 +248,101 @@ def replace_date(self):
     @property
     def refund_date(self):
         return self._refund_date
+
+class TicketType(enum.Enum):
```

Comment:
Just so you know, PEP8 really dislikes the way you use spaces.

![image](https://cloud.githubusercontent.com/assets/2474463/24841975/040dbd22-1d5e-11e7-9740-5b52bb28854e.png)